enhancements/machine-config/manage-boot-images.md

```diff
@@ -115,23 +119,37 @@ Any form factor using the MCO and `MachineSets` will be impacted by this proposa
 
 ##### Supported platforms
 
-The initial release(phase 0) will support GCP. In future releases, we will add in support for remaining platforms as we gain confidence in the functionality and understand the specific needs of those platforms. For platforms that cannot be supported, we aim to atleast provide documentation to perform the boot image updates manually. Here is an exhaustive list of all the platforms:
+The initial release(phase 0) will support GCP. In future releases, we will add in support for remaining platforms as we gain confidence in the functionality and understand the specific needs of those platforms. For platforms that cannot be supported, we aim to at least provide documentation to perform the boot image updates manually. Here is an exhaustive list of all the platforms:
 
-- gcp
 - aws
 - azure
-- alibabacloud
+- baremetal
+- gcp
+- ibmcloud
 - nutanix
-- powervs
 - openstack
+- powervs
 - vsphere
-- baremetal
-- libvirt
-- ovirt
-- ibmcloud
 
 This work will be tracked in [MCO-793](https://issues.redhat.com/browse/MCO-793).
 
+##### Projected timeline
+
+This is a tentative timeline, subject to change (GA = General Availability, TP = Tech Preview, EF = Default-on, Skew Enforced).
+
+| Platform | TP      | GA      | EF      |
+| -------- | ------- | ------- | ------- |
+| gcp      | 4.16    |4.17     |4.19     |
```

Comment:
nit: for historic/released entries, it would be nice to have links out to release notes like https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/release_notes/ocp-4-16-release-notes#ocp-4-16-machine-config-operator-boot-image_release-notes for this 4.16 functionality.  Or some way to see what here was guessing about the future, and what was reporting on the past?